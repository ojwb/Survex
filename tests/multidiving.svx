; pos=yes warn=1
*fix 1 0 0 0
*data diving station depth newline tape compass
1 0.0
  1.00 000
2 0.0 ; test
  1.00 090 ; test
3 0.0

3 1.0
  1.00 180
  ; testing
4 1.0
  ; testing
  1.00 270
5 1.0
  0.99 000
6 2.0
