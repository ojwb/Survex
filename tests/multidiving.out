./multidiving.svx:17:3: warning: Tape reading is less than change in depth
   0.99 000
   ^~~~

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 6 survey stations, joined by 5 legs.
There are 0 loops.
Total length of survey legs =    4.99m (   4.99m adjusted)
Total plan length of survey legs =    4.00m
Total vertical length of survey legs =    0.99m
Vertical range = 0.99m (from 6 at 0.99m to 5 at 0.00m)
North-South range = 1.00m (from 3 at 1.00m to 6 at -0.00m)
East-West range = 1.00m (from 4 at 1.00m to 2 at 0.00m)
   2 1-nodes.
   4 2-nodes.
There were 1 warning(s).
